client/src/AddVideo.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import './AddVideo.css';
+import axios from 'axios'; // Import axios for making HTTP requests
+
+function AddVideo({ onAdd }) {
+  const [title, setTitle] = useState('');
+  const [url, setUrl] = useState('');
+
+  
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+  
+    if (title && url) {
+      try {
+        // Create a new video object
+        const newVideo = {
+          title,
+          url,
+        };
+  
+        // Make a POST request to your server to add the video to the database
+        const response = await axios.post('http://localhost:7000/', newVideo);
+  
+        console.log("Server response:", response.data); // Debugging line
+  
+        // If the video is added successfully on the server, update the client-side state
+        if (response.status === 201) {
+          onAdd(response.data); // Assuming response.data contains the new video data
+          setTitle('');
+          setUrl('');
+  
+          // Reload the page
+          window.location.reload();
```

Comment:
You shouldn't need to reload pages in react apps - if you set new state, react will re-render the page for you if it needs it.